integration-tests/camel/invoked/root/camel-optaplanner/pom.xml

```diff
@@ -8,10 +8,6 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
 
-    <properties>
-        <skipTests>true</skipTests> <!-- Disabled because of https://github.com/apache/camel-quarkus/issues/2253 -->
```

Comment:
I think this needs to stay as skipped. IIRC the fix for this won't be available until we bump to Camel 3.9.x.

Reply:
Yeah, I also expected this to fail, but It passed for me locally. Let's see what Platform CI says.